### ISSUE TITLE
Datestamp/timestamp for Google Sheets integration

### ISSUE THREAD

**Issue by angieherrera** [verbb/formie#518]
In the Google Sheets integration, I'd love to have a column for the date/time stamp (in a human readable format) along the same lines as the Submission Title or ID. It would help with tracking of submissions outside of Craft.

**Comment by engram-design** [comment]
We'll look at adding a collection of useful variables to integrations, similar to the list available for email notifications and the variable-picker. For now, you'll need to add a Date field, with today's date as the default value and map that to the appropriate column in your sheet.

**Comment by chadcrowell** [comment]
Hi Josh - following up on this... the site in question I recently inherited from Angie. She did as you said and added a Date field with today's date as the default value. That is recording in Formie correctly with the submissions and I can see the value in the database column. It is mapped to a Google Sheet column on the integration, however the value of that date field is not populating in the Google sheet. Any reason a Date field would not pass through in the integration?

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.18](https://github.com/verbb/formie/releases/tag/1.4.18)
